src/main/resources/application.properties

```diff
@@ -137,3 +137,12 @@ management.health.elasticsearch.enabled=false
 management.health.status.order=DOWN, OUT_OF_SERVICE, UNKNOWN, UP, OUT_OF_SYNC, PROBLEM, COULD_NOT_CONNECT
 
 spring.banner.location=classpath:ccd-data-store-banner.txt
+
+# Hazelcast
+hazelcast.discovery.enabled=${HAZELCAST_DISCOVERY_ENABLED:true}
+hazelcast.discovery.consul.host=${HAZELCAST_DISCOVERY_CONSUL_HOST:localhost}
+hazelcast.discovery.consul.port=${HAZELCAST_DISCOVERY_CONSUL_PORT:8500}
+hazelcast.discovery.consul.healthy.only=${HAZELCAST_DISCOVERY_CONSUL_HEALTHY_ONLY:true}
+hazelcast.discovery.consul.delay.in.milliseconds=${HAZELCAST_DISCOVERY_CONSUL_DELAY_IN_MILLISECONDS:10000}
```

Comment:
I would suggest not to use unit of measurement in the property name. that is not inline with spring conventions. we can use comments as property descriptions including UoM's.